src/Backend/Core/Engine/Authentication.php

```diff
@@ -313,16 +316,40 @@ public static function loginUser(string $login, string $password): bool
     {
         $db = BackendModel::get('database');
 
-        // fetch the encrypted password
-        $passwordEncrypted = static::getEncryptedPassword($login, $password);
+        // build fallback for migrating devs
+        // todo remove in next major release
+        $encryptedPassword = BackendUsersModel::getEncryptedPassword($login);
```

Comment:
with an entry in the UPGRADE file yes, but maybe others have different opinions on this